### ISSUE TITLE
As a site owner I want my site to be WCAG AA compliant.

### ISSUE THREAD

**Issue by aweingarten** [Drupal4Gov/Drupal-GovCon-2017#75]

**Comment by onelittlebecca** [comment]
Oh GOOD! I was just thinking about this today. Glad this will be addressed.

**Comment by lkopacz** [comment]
@aweingarten I can also help out with this :)

**Comment by charlieartist** [comment]
@lkopacz I'm testing the colors through filters--please let me know if you see if anything needs updating. Also will be checking on type sizes, et. al. More wires/mocks should be up soon.

**Comment by lkopacz** [comment]
@charlieartist fabulous! I will keep you updated!

**Comment by jdearie** [comment]
Someone mentioned that they gray text on the home page may not have enough contrast.

**Comment by charlieartist** [comment]
@jdearie Just checked it here: http://webaim.org/resources/contrastchecker/ According to it, #999 (what is currently is) does fail. It should be adjusted to #666. I will make a note to do so. Thanks.
